.gitignore

```diff
@@ -6,6 +6,7 @@ _testing/
 *.lprof
 ext/GeoLite2-City.mmdb
 .env
+/ext/nginx.conf
```

Comment:
why /ext/ and not ext/

Reply:
(meaning my other ones should be updated)